scripts/upgrade

```diff
@@ -357,7 +357,7 @@ ynh_script_progression --message="Securing files and directories..."
 
 # Add a pids and socket directory for the systemd script.
 mkdir -p "$final_path/tmp/pids"
-mkdir "$final_path/tmp/sockets"
+mkdir -p "$final_path/tmp/sockets"
```

Comment:
`$final_path/tmp` is created the line before, so how come that changes anything...? :thinking:

Reply:
because when upgrading from  2.3.10~ynh1, `$final_path/tmp/sockets` is already present so it fails to create it and the upgrade fails :'(